src/helpers/assets.js

```diff
@@ -184,7 +184,7 @@ export const buildUniqueTokenList = (uniqueTokens, selectedShowcaseTokens) => {
   const showcaseTokens = [];
   const bundledShowcaseTokens = [];
 
-  const grouped = groupBy(uniqueTokens, token => token.collection.name);
+  const grouped = groupBy(uniqueTokens, token => token?.collection?.name);
```

Comment:
so what happens when a token doesn't have a collection name?

Reply:
I think this is a valid concern. We don't have this optional chaining on the web!

Reply:
@osdnk was an issue with previous local storage not having collection data. Reverted and bumped data version 💯